packages/testing/unit/npc-investment-manager-pnl.test.ts

```
@@ -0,0 +1,305 @@
+ * This test verifies the fix for the bug where realizedPnL was always returning 0
+ * instead of calculating the sum of realized PnL from closed positions.
+ */
+
+import { describe, expect, test, beforeAll, afterAll } from 'bun:test';
+import {
+  db,
+  actorState,
+  poolPositions,
+  perpPositions,
+  eq,
+  sql,
```

Biome (2.3.13)

[error] Several of these imports are unused.

Unused imports might be the result of an incomplete refactoring.
Unsafe fix: Remove the unused imports.


(lint/correctness/noUnusedImports)